content/en/docs/reference/setup-tools/kubeadm/kubeadm-init.md

```diff
@@ -207,7 +208,13 @@ instance is upgraded. The deprecated `UpgradeAddonsBeforeControlPlane` feature g
 behavior. You should not need this old behavior; if you do, you should consider changing your cluster or upgrade processes, as this
 feature gate will be removed in a future release.
 
-List of removed feature gates:
```

Comment:
Instead of removing this line, I'd suggest we move it before the table below.

Reply:
yes, will revert it, just cherry-picked the change from https://github.com/kubernetes/website/pull/43510.